android/services/mongodb-remote/src/main/java/com/mongodb/stitch/android/services/mongodb/remote/Sync.java

```diff
@@ -43,7 +45,7 @@
    *                         document.
    * @param errorListener the error listener to invoke when an irrecoverable error occurs
    */
-  void configure(final ConflictHandler<DocumentT> conflictHandler,
+  void configure(@NonNull final ConflictHandler<DocumentT> conflictHandler,
                  final ChangeEventListener<DocumentT> changeEventListener,
```

Comment:
[nit] if we're labeling the conflict handler `@NonNull`, I guess we should explicitly mark the rest as `@Nullable`?